packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -772,7 +772,7 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
                     this.logger.sendErrorEvent({ eventName: "fetchSnapshotError", sequenceNumber, seqNumberFromOps });
                     value.sequenceNumber = undefined;
                 } else if (canCache) {
-                    this.cache.persistedCache.put(
+                    this.epochTracker.put(
```

Comment:
This is not required and is noop.
If you trace code up to OdspDocumentService.create() call, you will see that both are coming from createOdspCacheAndTracker() and are the same object.

Basically there is no way to access "raw" cache object, you always access the one provided by EpochTracker that handles versions, epochs, etc..

I like separation into two objects / interfaces (that's the reason I've left this.cache.persistedCache at all) - i.e. for interface to have single responsibility, what's missing is defining interface for epochTracker (i.e. not using class here) and stripping from it cache methods, such it's a bit more clear in code like here that epochTacker interface has only one purpose. I did not finish this part in my prior refactoring - feel free to add :)

Reply:
I don't think putting value in cache object is storing epoch value with it.

Reply:
oh ok. We are putting epoch tracker in cache. Got it.

Reply:
Closing this PR.